app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    cmd, source_file, des = command.split()
+
+    if cmd != "mv" or len([cmd, source_file, des]) != 3:
+        raise ValueError("Invalid command")
+
+    # Check if the destination starts with "dir" and if "dir" exists, remove it
+    if des.startswith("dir") and os.path.exists("dir"):
+        shutil.rmtree("dir")
```

Comment:
Don't understand, why this check is needed

Reply:
you are right, it's not needed.